wikimedia/wikipedia-ios
# Add banner to subscription action

**Phabricator:** https://phabricator.wikimedia.org/T311639

### Notes
* This task adds a banner to the subscription action
* We are not calling the API at this moment (but methods were added to the view model)
* The layout differs from figma, as we are using an existing component at the moment

### Test Steps
1. Go to a talk page, and click the subscribe button. Make sure the alert shows at the bottom.
2. Click unsubscribe, verify the banner
